sa/model_test.go

```diff
@@ -245,17 +245,20 @@ func TestCerficatesTableContainsDuplicateSerials(t *testing.T) {
 	test.AssertNotError(t, err, "couldn't insert valid certificate")
 
 	// This should return the certificate that we just inserted.
-	_, err = SelectCertificate(sa.dbMap, serialString)
+	certA, err := SelectCertificate(sa.dbMap, serialString)
 	test.AssertNotError(t, err, "received an error for a valid query")
 
 	// Insert a certificate with a serial of `1337` but for a different
 	// hostname.
 	err = insertCertificate(sa.dbMap, fc, "1337.net", "leet", 1337, 1)
 	test.AssertNotError(t, err, "couldn't insert valid certificate")
 
-	// With a duplicate being present, this should error.
-	_, err = SelectCertificate(sa.dbMap, serialString)
-	test.AssertError(t, err, "should've received an error for multiple rows")
+	// Despite a duplicate being present, this shouldn't error.
+	expectA, err := SelectCertificate(sa.dbMap, serialString)
```

Comment:
this is very much a nit, but this should be named `certB` or `certDupe` or something like that. It's not `expectA`, because it's a real result we're getting from exercising the code under test, not a hard-coded expected value.